src/DataCollectors/TraceDataCollector/Microsoft.VisualStudio.TraceDataCollector.csproj

```diff
@@ -19,7 +19,7 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.Internal.CodeCoverage">
```

Comment:
nicpick. We can inline `Version` here to make it more consistent.

Reply:
changed